frontend/src/components/object/ObjectProperties.vue

```diff
@@ -1,13 +1,55 @@
 <script setup lang="ts">
+import { onMounted, ref, watch } from 'vue';
+import { storeToRefs } from 'pinia';
+import { useBucketStore, useUserStore } from '@/store';
+import GridRow from '@/components/form/GridRow.vue';
+import { RouteNames } from '@/utils/constants';
```

Comment:
Style: Import ordering should be done based on library name. Newline between external and internal deps.
```ts
import { storeToRefs } from 'pinia';
import { onMounted, ref, watch } from 'vue';

import GridRow from '@/components/form/GridRow.vue';
import { useBucketStore, useUserStore } from '@/store';
import { RouteNames } from '@/utils/constants';
```